lib/solidus_subscriptions/subscription_line_item_builder.rb

```diff
@@ -9,8 +9,6 @@ def create_subscription_line_item(line_item)
         subscription_params.merge(spree_line_item: line_item)
       )
 
-      # Rerun the promotion handler to pickup subscription promotions
-      ::Spree::PromotionHandler::Cart.new(line_item.order).activate
```

Comment:
I guess my only concern is this. Is this safe? Do we need to do something _different_ to handle legacy versus new promotions engines?

Reply:
@jarednorman Man, like really, from the bottom of my heart, could you be more specific, I am sure when you write this kind of comments you already have a very specific way in mind to do it.

Reply:
@jarednorman that is actually a tangible point and helps much more than the previous comment!!!

Reply:
@jarednorman hey, sorry for replying so late. The issue of the removal of `::Spree::PromotionHandler::Cart.new(line_item.order).activate` was discussed here: https://github.com/gms-electronics/solidus_subscriptions/pull/1 If you have something else to consider please tell us.